src/app/register/pages/register/register.component.ts

```diff
@@ -209,21 +212,23 @@ export class RegisterComponent implements OnInit, AfterViewInit {
 
     if (this.FormGroupStepA.valid) {
       this.personalData = this.FormGroupStepA.value.personal
+      const searchValue =
+        this.personalData.familyNames.value +
+        ' ' +
+        this.personalData.givenNames.value
+      const searchParams: SearchParameters = {}
+      searchParams.searchQuery = searchValue
 
-      this._register
-        .checkDuplicatedResearcher({
```

Comment:
Maybe we can delete `checkDuplicatedResearcher`

Reply:
There is another card for cleanup @leomendoza123 https://trello.com/c/sDQrtQuI/781-remove-code-related-to-dupicateresearcherjson